server/src-lib/Hasura/SQL/DML.hs

```diff
@@ -209,6 +209,9 @@ incOp = SQLOp "+"
 mulOp :: SQLOp
 mulOp = SQLOp "*"
 
+jsonbGetOp :: SQLOp
+jsonbGetOp = SQLOp "->"
```

Comment:
Use jsonb operator `#>` to extract at specified path. You can take help of `SEArray` constructor of `SQLExp` data type in `Hasura/SQL/DML.hs` module to generate array expression on RHS.